docs/cookbook/extending-product-list.md

```diff
@@ -0,0 +1,129 @@
+# Extending Product List
+
+Frontend product lists leverage [the read model concept](/docs/model/introduction-to-read-model.md), i.e. special view objects are used instead of common Doctrine entities in the Twig templates.
```

Comment:
I'm missing sort of motivation... why should I read this? Something like "You need to add more information to product lists but adding them into entities doesn't work? That is because..."

The formulation is strange, but I would like to see a motivation, what problem this cookbook solves.